src/util/hidepassword.ts

```diff
@@ -0,0 +1,14 @@
+export class HidePassword {
+
+    static hideTokenpassword(value: string) {
+        const regex = /--token\s*=\s*([^\s]*)/;
+        const tokenRegex = value.match(regex);
+        return (tokenRegex) ?  value.replace(tokenRegex[1], '**********') : value;
```

Comment:
should be as simple as:
``` JavaScript
 return value.replace(/--token\s*=\s*([^\s]*)/], '**********') ;
```

Reply:
I am matching the string and replacing the password on a group-wise. if I just use the regex here it will replace the password for token
 ```oc login https://192.168.64.82:8443 --token=nLKyqdaaSii54GZWHJQVeO_fVEPK8rgve6warak_yA0 -> oc login https://192.168.64.82:8443 *******``` 
and for credential 
``` `login https://192.168.64.74:8443 -u abc -p abc --insecure-skip-tls-verify`  -> `login https://192.168.64.74:8443 -u ***(which is abc) -p abc --insecure-skip-tls-verify` ```

 ```